### PR TITLE
[CLI] bump CLI version to 1.0.14 for release

### PR DESCRIPTION
**Pull request by 0xjinn**

### CODE REVIEW

File: crates/aptos/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Aptos CLI Changelog
 
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
-## [1.0.14] - TBD
+## [1.0.14] - 2023/05/26
 ### Added
-* update DB boostrap command with new db restore features
+- Add nested vector arg support
+- update DB boostrap command with new db restore features
```
Comment (banool) [inline crates/aptos/CHANGELOG.md:7]:
```suggestion
- Updated DB bootstrap command with new DB restore features
```

Comment (0xjinn) [reply crates/aptos/CHANGELOG.md:7]: Good catch on grammar!

**Review by banool — APPROVED** [review]
Send it!!!!!!!

**Review by msmouse — APPROVED** [review]
Thanks

**Review by areshand — APPROVED** [review]

**Pull request merged**